app/views/stages/_fields.html.erb

```diff
@@ -116,14 +116,19 @@
 
   <div id="commands">
     <%= form.collection_check_boxes(:command_ids, form.object.all_commands, :id, :command) do |b| %>
+      <% command = b.object %>
+      <% others = (command.stages - [@stage]).map(&:name) %>
```

Comment:
I don't think this makes it any simpler ... it's and array an we do some simple array math :)

```Ruby
def others(stage)
  stages - [stage]
end
```

`command.others(stage)` vs `command.stages - [@stage]`